src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -54,6 +59,12 @@ public Swerve(SwerveConstants constants, Modules modules, ISwerveGyro gyro) {
         this.gyroInputs = new SwerveGyroInputsAutoLogged();
         this.currentAngleSupplier = this::getAbsoluteHeading;
 
+        this.stateHelper = new SwerveStateHelper(
+                () -> new Pose2d(0,0, currentAngleSupplier.get()), //default pose estimator
```

Comment:
```suggestion
                () -> new Pose2d(0, 0, currentAngleSupplier.get()), //default pose estimator
```
what will it execute amp aim assist, but other aimassists no?

Reply:
yes cuz if empty return;

Reply:
but it will do the speaker cuz it thinks it got a position. when we will have the default pose est it will be good

Reply:
thats not good if it do speaker, it need to know what it can do and do only 
maybe do optional pose and rotation supplier